payments-core/src/main/java/com/stripe/android/PaymentBrowserAuthStarter.kt

```diff
@@ -16,19 +16,15 @@ internal interface PaymentBrowserAuthStarter :
     AuthActivityStarter<PaymentBrowserAuthContract.Args> {
     class Legacy(
         private val host: AuthActivityStarterHost,
-        private val hasCompatibleBrowser: Boolean,
         private val defaultReturnUrl: DefaultReturnUrl
     ) : PaymentBrowserAuthStarter {
         override fun start(args: PaymentBrowserAuthContract.Args) {
             val extras = args
                 .copy(statusBarColor = host.statusBarColor)
                 .toBundle()
 
-            val shouldUseBrowser =
-                hasCompatibleBrowser && args.hasDefaultReturnUrl(defaultReturnUrl)
-
             host.startActivityForResult(
-                when (shouldUseBrowser) {
```

Comment:
does this change imply that even non Chrome browsers are able to handle [CustomTabsIntent](https://github.com/stripe/stripe-android/blob/3856082ff2fadc504734910b3a93e066b9829a8a/payments-core/src/main/java/com/stripe/android/payments/StripeBrowserLauncherViewModel.kt#L49) ?

Reply:
It is both true that non-Chrome browsers can support Custom Tabs, and that we should use the default device browser if Custom Tabs are not available.